### ISSUE TITLE
The button to open the ‘Keyboard Controls’ modal dialog is visually missing. 

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5211]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

This could be problematic, especially for those using speech recognition, or other AT’s. Providing a properly labeled visible button will improve way finding and reduce cognitive load. 

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=true 

### How to reproduce the issue?

1. Open [https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=true](https://wwwppe.microsoft.com/en-us/surface) in Edge. 
2. [Tab] to the ‘Help me choose’ button and open the window. 
3. Without tabbing, can you see that a list of keyboard controls is available? 
4. Continue tabbing until the ‘Keyboard Controls’ modal dialog opens, then dismiss the dialog and continue through the interface. 
5. Once you are interacting with the prompts, can you easily find where to access or remember how to access the ‘Keyboard Controls’ modal?

### What do you expect?

Provide a visible button to allow users to open the ‘Keyboard Controls’ modal dialog.

### What actually happened?

The button to open the ‘Keyboard Controls’ modal dialog is visually missing. 

### Do you have any screenshots or recordings to repro the issue?

![image](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/36c31a60-6483-429c-9635-5c380e2b4178)

### MAS reference

https://aka.ms/MAS1.3.1

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;
